### DISCUSSION TITLE
火炬之光

### DISCUSSION THREAD

**Discussion by Gui-Cang** [Gamuxorg/bbs#323] [游戏或网站运行反馈]
### 以下是推荐模板
游戏名称：火炬之光

操作系统：mint 20 ，桌面是肉桂

bug描述：无法打开Appimage，而同时下载的盐和避难所可以

终端运行输出 ：*（如果有的话）*

自行尝试解决bug的方式和结果：*（如果有的话）*

**Comment by hyh19962008** [comment]
可以提供一下终端运行输出的信息吗

**Comment by Gui-Cang** [comment]
/tmp/.mount_torchlfl7Gkz/usr/bin/Torchlight.bin.x86_64: error while loading shared libraries: libfreeimage.so.3: cannot open shared object file: No such file or directory
应该是库或者挂载的时候出问题了

**Comment by hyh19962008** [comment]
sudo apt install libfreeimage3

Best

________________________________

**Comment by Gui-Cang** [comment]
> sudo apt install libfreeimage3 Best
> […](#)
> ________________________________ From: Gui-Cang <notifications@github.com> Sent: Sunday, December 13, 2020 3:03:51 PM To: Gamuxorg/bbs <bbs@noreply.github.com> Cc: Demon <hyh19962008@gmail.com>; Comment <comment@noreply.github.com> Subject: Re: [Gamuxorg/bbs] 火炬之光 (#130) /tmp/.mount_torchlfl7Gkz/usr/bin/Torchlight.bin.x86_64: error while loading shared libraries: libfreeimage.so.3: cannot open shared object file: No such file or directory 应该是库或者挂载的时候出问题了 ― You are receiving this because you commented. Reply to this email directly, view it on GitHub<[#130 (comment)](https://github.com/Gamuxorg/bbs/issues/130#issuecomment-743964347)>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AFRK4ULELN7EBORCKKFUMLTSURRNPANCNFSM4UX7ZG3Q>.

多谢老哥，可以打开了。就是窗口是固定大小，无法缩放。已经很感谢了。
Thx, it works. But we can't change the window's size. It sets  fixed size. Still thanks a lot.

**Comment by shouhuanxiaoji** [comment]
火炬之光1的问题可能是由于，官方使用了一个定制的sdl，我再看看

**Comment by shouhuanxiaoji** [comment]
做了一个新版本，测试可以修改分辨率，可以试试
